astronomer/providers/apache/hive/sensors/hive_partition.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, Dict, Optional
+
+from airflow import AirflowException
+from airflow.providers.apache.hive.sensors.hive_partition import HivePartitionSensor
+
+from astronomer.providers.apache.hive.triggers.hive_partition import (
+    HivePartitionTrigger,
+)
+
+
+class HivePartitionSensorAsync(HivePartitionSensor):
+    """
+    Waits for a given partition to show up in Hive.
+    Note: HivePartitionSensorAsync uses implya library instead of pyhive
+    since pyhive is currently unsupported. Refer https://github.com/dropbox/PyHive.
+    Since we use implya library, please set the connection to use the port 10000 instead of 9083
+
```

Comment:
can you also mention the limitations of this sensor, some thing like we can currently use it for `auth_mechanism PLAIN` only?